scripts/_common.sh

```diff
@@ -6,55 +6,35 @@
-    ynh_replace_string --match_string __DB_PWD__ --replace_string "$db_pwd" --target_file "$final_path/horde/config/conf.php"
-    ynh_replace_string --match_string __ADMIN_USER__ --replace_string "$admin" --target_file "$final_path/horde/config/conf.php"
-    ynh_replace_string --match_string __SECRET_KEY__ --replace_string "$secret_key" --target_file "$final_path/horde/config/conf.php"
-    
-    cp ../conf/horde_imp_conf.php "$final_path/horde/imp/config/conf.php"
-    
-    cp ../conf/horde_registry.php "$final_path/horde/config/registry.local.php"
-    ynh_replace_string --match_string __PATH__ --replace_string "$path_url" --target_file "$final_path/horde/config/registry.local.php"
-
-    cp ../conf/gollem_backends.php "$final_path/horde/gollem/config/backends.local.php"
-    ynh_replace_string --match_string __GOLLEM_DATA_DIR__ --replace_string "$gollem_data_dir" --target_file "$final_path/horde/gollem/config/backends.local.php"
-
-    cp ../conf/ingo_backends.php "$final_path/horde/ingo/config/backends.local.php"
-
-    ynh_store_file_checksum --file "$final_path/horde/config/conf.php"
-    ynh_store_file_checksum --file "$final_path/horde/imp/config/conf.php"
-    ynh_store_file_checksum --file "$final_path/horde/config/registry.local.php"
-    ynh_store_file_checksum --file "$final_path/horde/gollem/config/backends.local.php"
-    ynh_store_file_checksum --file "$final_path/horde/ingo/config/backends.local.php"
+    ynh_backup_if_checksum_is_different --file "$install_dir/horde/config/conf.php"
+    ynh_backup_if_checksum_is_different --file "$install_dir/horde/imp/config/conf.php"
+    ynh_backup_if_checksum_is_different --file "$install_dir/horde/config/registry.local.php"
+    ynh_backup_if_checksum_is_different --file "$install_dir/horde/gollem/config/backends.local.php"
+    ynh_backup_if_checksum_is_different --file "$install_dir/horde/ingo/config/backends.local.php"
+
+
+    ynh_add_config --template="horde_conf.php" --destination="$install_dir/horde/config/conf.php"
+    ynh_add_config --template="/horde_imp_conf.php" --destination="$install_dir/horde/imp/config/conf.php"
+    ynh_add_config --template="horde_registry.php" --destination="$install_dir/horde/config/registry.local.php"
+    ynh_add_config --template="gollem_backends.php" --destination="$install_dir/horde/gollem/config/backends.local.php"
+    ynh_add_config --template="ingo_backends.php" --destination="$install_dir/horde/ingo/config/backends.local.php"
+
+    ynh_store_file_checksum --file "$install_dir/horde/config/conf.php"
+    ynh_store_file_checksum --file "$install_dir/horde/imp/config/conf.php"
+    ynh_store_file_checksum --file "$install_dir/horde/config/registry.local.php"
+    ynh_store_file_checksum --file "$install_dir/horde/gollem/config/backends.local.php"
+    ynh_store_file_checksum --file "$install_dir/horde/ingo/config/backends.local.php"
```

Comment:
```suggestion
    ynh_add_config --template="horde_conf.php" --destination="$install_dir/horde/config/conf.php"
    ynh_add_config --template="/horde_imp_conf.php" --destination="$install_dir/horde/imp/config/conf.php"
    ynh_add_config --template="horde_registry.php" --destination="$install_dir/horde/config/registry.local.php"
    ynh_add_config --template="gollem_backends.php" --destination="$install_dir/horde/gollem/config/backends.local.php"
    ynh_add_config --template="ingo_backends.php" --destination="$install_dir/horde/ingo/config/backends.local.php"
```

Not sure what's the point of calling `ynh_backup_if_checksum_is_different` and `ynh_store_file_checksum` manually, it's already done by `ynh_add_config`